pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -176,3 +180,8 @@ message TLSClientCertificateAuthenticationPolicy {
   // validate the remote TLS client.
   string client_certificate_authorities = 1;
 }
+
+message JWTAuthenticationPolicy {
+  // Key to use to authenticate the JWTs.
```

Comment:
Could we describe what the intended format of this is? Maybe state that this needs to be a PEM file.

Also, this only adds support for symmetric keys, right? Could we also add support for asymmetric ones? In my opinion any sane deployment should use the latter.

Reply:
Will update the docs. This PR supports symmetric keys and public keys. The JWK format can encode both kinds of keys. Moreover, the code used to read the key can decode JWK, X509, and PEM formats.